AutoRest/Generators/Ruby/Azure.Ruby/Templates/PageModelTemplate.cshtml

```diff
@@ -0,0 +1,94 @@
+@using System.Linq
+@using System.Collections.Generic
+@using Microsoft.Rest.Generator.ClientModel
+@using Microsoft.Rest.Generator.Ruby
+@using Microsoft.Rest.Generator.Azure.Ruby
+@using Microsoft.Rest.Generator.Azure.Ruby.Templates
+@using Microsoft.Rest.Generator.Ruby.TemplateModels
+@using Microsoft.Rest.Generator.Utilities
+@inherits Microsoft.Rest.Generator.Template<PageTemplateModel>
+# encoding: utf-8
+@Header("# ")
+@EmptyLine
+module @(Settings.Namespace)
+  module Models
+    #
+    @WrapComment("# ", string.IsNullOrEmpty(Model.Documentation) ? "Model object." : Model.Documentation)
+    #
+    class @Model.Name@(Model.GetBaseTypeName())
+    @if (Model.Includes.Any())
+    {
+      @EmptyLine
+      foreach (var include in Model.Includes)
+      {
+      @:include @include
+      }
+      @EmptyLine
+    }
+
+    @if (Model.IsPolymorphic && Model.BaseModelType == null)
+    {
+      @:@@@@discriminatorMap = Hash.new
+      @:@@@@discriminatorMap["@Model.SerializedName"] = "@Model.Name.ToLowerInvariant()"
```

Comment:
value should be @Model.Name instead of @Model.Name.ToLowerInvariant(). This is regarding the bug i found. PR is [https://github.com/Azure/autorest/pull/1245](https://github.com/Azure/autorest/pull/1245)